libr/bin/p/bin_mach0.c

```diff
@@ -503,6 +507,180 @@ static RBinInfo *info(RBinFile *bf) {
 	return ret;
 }
 
+static bool init_patch_reloc(ut32 cpu_type, struct reloc_t * reloc, ut64 * pc, ut64 * ins_len) {
```

Comment:
Ok sorry :D probably the double switch was just clearer than this... don't hate me :)

Reply:
yeah i was thinking the same

Reply:
got rid of that refactoring, i think we can refactor this when/if it gets more complex than that in the future.

Reply:
Yep, that's fine!